units/host_status.go

```diff
@@ -53,8 +54,11 @@ func makeCloudHostReadyJob() *cloudHostReadyJob {
 }
 
 func (j *cloudHostReadyJob) Run(ctx context.Context) {
+	// Jobs never appear to exceed 1 minute, but add a bunch of padding.
+	const timeout = 10 * time.Minute
```

Comment:
set TimeInfo.MaxTime in the job's (public) constructor.

Reply:
Done.